app/grandchallenge/core/templatetags/bleach.py

```diff
@@ -28,14 +28,30 @@ def clean(html: str):
 
 
 @register.filter
-def md2html(markdown: str | None, link_blank_target=False):
+def md2html_email(markdown: str | None):
```

Comment:
Do we need another filter or is this possible to achieve with just the options?

Reply:
It's possible with the options, but since they are booleans it gets verbose really quick to keep it readable:
```Jinja
{% set args = { 'option_a': True, 'option_b': False } %}
{% body | md2html(args)
```

Since this specific function needs to be called at two places, we'd need to repeat the exact pattern at both places. 

Alternatively, we could add a single boolean switch to mark it an email that is to be rendered...